### PR TITLE
JAMES-2822 Document usersrepository.xml options

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: src/site/xdoc/server/config-users.xml [hunk 1]
```diff
@@ -26,19 +26,32 @@
...
+
+    <subsection name="General configuration">
+
+      <p>All Users Repositories provide at least these two options</p>
+
+      <dl>
+        <dt><strong>enableVirtualHosting</strong></dt>
+        <dd>true or false. Add domain support for users (default: false, except for Cassandra Users Repository)</dd>
+        <dt><strong>administratorId</strong></dt>
+        <dd>user's name. Allow an user to acces to the <a href="https://tools.ietf.org/html/rfc4616#section-2">impersonation command</a>, acting on the behalf of any user.</dd>
```
Comment (mbaechler) [inline src/site/xdoc/server/config-users.xml]: name or email address?

Comment (trantienduchn) [inline src/site/xdoc/server/config-users.xml]: s/acces/access

Comment (trantienduchn) [reply src/site/xdoc/server/config-users.xml]: is this option mandatory?

Comment (blackheaven) [reply src/site/xdoc/server/config-users.xml]:
```
    @Override
    public boolean isAdministrator(String username) throws UsersRepositoryException {
        if (administratorId.isPresent()) {
            return administratorId.get().equals(username);
        }
        return false;
    }
```

I would say no

Comment (blackheaven) [reply src/site/xdoc/server/config-users.xml]: it's optional

**Comment by blackheaven** [comment]
Screen

Screenshot

**Comment by mbaechler** [comment]
merged

**Pull request closed without merging**